src/System/Wlog/Handler.hs

```diff
@@ -63,10 +66,11 @@ class LogHandler a where
     -- given by the most recent call to 'setLevel'.
     handle :: a -> LogRecord -> LoggerName -> IO ()
```

Comment:
Well, this function came from `hslogger`. It's really strange that it has default implementation and it's never implemented by any logger. Could you please move this function from this type class to top level? And rename it... `handle` name is really toooooo generic and awkward. Probably something like `outputMessage` or `logHandlerMessage` might be better.

Also, one more thing I don't like in current design of `log-warper`: too many intermediate calls between something meaningful actually happens in `handle`. Let me explain:

1. Current `logMessage` is two-lines function which calls `dispatchMessage`.
```haskell
logMessage severity t = do
    name <- askLoggerName
    dispatchMessage name severity t
```
2. `dispatchMessage` for `IO` is one line function which just calls `logM`.
```haskell
dispatchMessage name prior msg = logM name prior msg
```
3. `logM` function is two-lines function which just calls `logL`:
```haskell
logM logname pri msg = do
    l <- getLogger logname
    logL l pri msg
```
4. `logL` is one line function which finally calls `handle`:
```haskell
logL l pri msg = handle l (LR pri msg) (const True)
```

I think this can be done better... We can remove `logMessage` from this chain because this function should remain polymorphic. But the others... Unless functions from this chain are not used anywhere else, we can remove intermediate redundant functions.

Also, I think function like `debugM` are not needed anymore:
https://github.com/serokell/log-warper/blob/4a0dc0922f7f487fe35f054169a56aeee923afff/src/System/Wlog/Logger.hs#L161

Reply:
I agree, too many functions, but here I see that only `logL` could be thrown away, cause it's the only usage for the function. Also there is `logLCond`, with the same issue..
I guess others should stay :thinking: